beacon-chain/state/state-native/state_trie.go

```diff
@@ -650,6 +650,11 @@ func InitializeFromProtoUnsafeFulu(st *ethpb.BeaconStateFulu) (state.BeaconState
 	for i, v := range st.ProposerLookahead {
 		proposerLookahead[i] = primitives.ValidatorIndex(v)
 	}
+	// Proposer lookahead must be exactly 2 * SLOTS_PER_EPOCH in length. We fill in with zeroes instead of erroring out here
+	for i := len(proposerLookahead); i < 2*fieldparams.SlotsPerEpoch; i++ {
```

Comment:
do we not have a param value for this? i think i saw one for the ssz protobuf

Reply:
I think it's supposed to be `params.BeaconConfig().MinSeedLookahead + 1`, not just `2`, although it is `2` in both cases.